csharp-design-patterns/UnitOfWork/UnitOfWork/Program.cs

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
This blank line at the beginning of the file should be removed.

Reply:
removed